### DISCUSSION TITLE
Issue with S3 Generic and openstack

### DISCUSSION THREAD

**Discussion by robertkruck** [requarks/wiki#5431] [Help / Questions]
I'm having a little issue getting s3 generic storage to work with our openstack environment.  The error is "self signed certificate in certificate chain"  I have wiki.js set to use a wildcard cert and that works fine accessing the wiki over https.  I'm not sure if this is a wiki.js issue or I just need to install the CA-chain certs on the host/docker containers.

**Comment by dumbasPL** [comment]
have you managed to figure it out? I have MinIO setup behind a reverse proxy using a wildcard cert from Let's Encrypt and for me, the error only says `self-signed certificate`. The certificate is NOT self-signed and works fine with other software and browsers.
